bump_rpm.sh

```diff
@@ -104,7 +104,7 @@ if [[ $CURRENT_VERSION != $NEW_VERSION ]] ; then
 		rm -rf "$UNPACKED_GEM_DIR"
 	fi
 
-	git commit -m "Update $PACKAGE_NAME to $NEW_VERSION"
+	[[ "$SKIP_GIT_COMMIT" == "1" ]] || git commit -m "Update $PACKAGE_NAME to $NEW_VERSION"
```

Comment:
I'd prefer an `if` statement

Reply:
That would allow any non empty value, right? don't we want to force it to be `1`?
I've used test != 1, but if you feel non-empty value is more readable, I'll use yours :shrug:

Reply:
Well, I'm fine with either check. I would like to avoid `[[ ]] || command` since I think it's a bad pattern.